src/features/security-cloud-game/SecurityCloudGame.scss

```diff
@@ -22,13 +22,13 @@
     }
 }
 
-.box-ui-security-cloud-game {
+.bdl-SecurityCloudGame {
```

Comment:
I think we talked about this but to double check, this specific component (the version in this repo i.e. importing from `box-ui-elements/src/features/security-cloud-game/`) is not used in any parent app correct? updating this class name will not impact any existing page / app because it's not being used anywhere.

the only "version" of this component in production is being imported from a different repo?

Reply:
No, it's not used in any parent app. And changing class names won't affect any existing page or app.

Yes, it was processed and uploaded to production in a bundle through an archived/deprecated repo a long time ago, and it had not been updated for at least 3 years now.